webapp/packages/core-root/src/SessionEventSource.ts

```diff
@@ -135,6 +137,26 @@ export class SessionEventSource implements IServerEventEmitter<ISessionEvent, IS
 
   disconnect() {
     this.disconnected = true;
+
+    if (!this.subject.closed) {
+      this.subject.unsubscribe();
+    }
+
+    if (!this.closeSubject.closed) {
+      this.closeSubject.unsubscribe();
+    }
+
+    if (!this.openSubject.closed) {
+      this.openSubject.unsubscribe();
+    }
+
+    if (!this.oldEventsSubject.closed) {
+      this.oldEventsSubject.unsubscribe();
+    }
+
+    if (!this.errorSubject.closed) {
+      this.errorSubject.unsubscribe();
+    }
```

Comment:
this will lead to dead `SessionEventSource` and all it's subscriptions.
What if we want to pause and resume websocket?